client/src/components/application/Examine/CompName.vue

```diff
@@ -485,7 +457,9 @@
         this.is_making_decision = false;
       },
       setFocus: function() {
-        this.$refs.search.focus();
+        if(this.$refs.search) {
```

Comment:
threw this in here because I was noticing that when the page loaded for a completed NR, the mounted() method is still telling the search field to take focus.  But there is no search field if the NR being displayed is in a completed state.  :-\